react-multiple-pages/my-app/src/pages/NotFound.tsx

```diff
@@ -0,0 +1,14 @@
+import { Link } from 'react-router-dom';
+
+export function NotFound() {
+  return (
+    <div className="flex">
+      <Link to="/" className="flex-1 py-12 text-center">
```

Comment:
I think this Link should have replaced the div that surrounds "Return to Dashboard" instead of the entire text.